docs/administration/security/blocklist.md

```diff
@@ -1,27 +1,57 @@
 # Plugin Blocklist
 
-The Plugin Blocklist can be used when plugins need to be blocked or prevented from being installed or used for security or other use case situations.  To use the plugin blocklist, you will need to create a blocklist yaml file in the following format:
+The Plugin Blocklist can be used when plugins need to be blocked or prevented from being installed or used for security or other use-case situations.  To use the plugin blocklist, you will need to create a blocklist yaml file in the following format:
 
 ```yaml
 fileNameEntries:
-    - rundeckpro-plugin-jar-regex
-    - rundeckpro-plugin-jar-regex
-    - rundeckpro-plugin-jar-regex
-
+    - rundeckpro-plugin-jar-name
+    - rundeckpro-plugin-jar-name
+    - rundeckpro-plugin-jar-name
 providerNameEntries:
    ProviderService:
       - provider-name
       - provider-name
+```
+:::tip
+If you intend to use the fileNameEntries or providerNameEntries, both of these properties must be declared otherwise Rundeck will fail to start.
```

Comment:
Are you trying to say that "even if one section is blank, it must still be declared" ?

Reply:
Yes, that is what I was trying to explain with this paragraph, I will fix it with your suggestion.